src/clientiface.cpp

```diff
@@ -250,7 +267,9 @@ void RemoteClient::GetNextBlocks (
 			*/
 
 			f32 dist;
-			if (!isBlockInSight(p, camera_pos, camera_dir, camera_fov, d_blocks_in_sight, &dist)) {
+			if (!(isBlockInSight(p, camera_pos, camera_dir, camera_fov, d_blocks_in_sight, &dist) ||
+			      (playerspeed.getLength() > 1.0*BS &&
```

Comment:
specify float and spaces around '*'
1,0f * BS

Reply:
\*  `1.0f * BS` (dot instead of comma)

Also, spaces instead of tab for the indent.

Reply:
I wanted to make the logic visible from the indentation. Tabs look weird here, but in any case, I'll change it.

Reply:
Oops yes dot.

Reply:
Either 1 or 2 tabs of indent are ok, i would prefer 1 to reduce the chance of a overly-long line.